docker/nugus_sim/run.py

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env python3
+
+import os
+import subprocess
+import sys
+from typing import Tuple
+
+import ruamel.yaml
+
+BINARIES_DIR = "/home/nubots/NUbots/binaries"
+CONFIG_DIR = BINARIES_DIR + "/config"
+
+ENV_VARS = ["ROBOCUP_ROBOT_ID", "ROBOCUP_TEAM_COLOR", "ROBOCUP_SIMULATOR_ADDR"]
+
+
+def read_args() -> Tuple[str, dict]:
+    if len(sys.argv) != 2:
+        print("Please specify a single role to run!")
+        sys.exit(1)
+
+    role = sys.argv[1]
+
+    built_roles = []
+
+    # Built roles
+    for filename in os.listdir(BINARIES_DIR):
+        if os.path.isfile(os.path.join(BINARIES_DIR, filename)):
+            built_roles.append(filename)
+
+    if role not in built_roles:
+        print("The role '" + role + "' does not exist!")
+        sys.exit(1)
+
+    # Read env vars
+    config = {var: os.environ.get(var) for var in ENV_VARS if var in os.environ}
+
+    # List of env vars that didn't get set :(
+    unset_vars = [var for var in ENV_VARS if var not in config]
+
+    # If not all needed environment variables were set
+    if len(unset_vars) != 0:
+        print("The following environment variables were not set!")
+        for var in unset_vars:
+            print(var)
+        sys.exit(1)
+
+    return role, config
+
+
+def set_env_vars(config: dict) -> None:
+    yaml = ruamel.yaml.YAML()
+    os.chdir(CONFIG_DIR)
+
+    # ROBOCUP_ROBOT_ID
+    global_config_fname = "GlobalConfig.yaml"
+    with open(global_config_fname) as file:
+        global_config_data = yaml.load(file)
```

Comment:
`PyYAML` provides a `safe_load` function, if we need to use `ruamel.yaml` for whatever reason we should make sure we use a similar function
```suggestion
    with open(os.path.join(CONFIG_DIR, "GlobalConfig.yaml") as file:
        global_config_data = yaml.safe_load(file)
```